instrumentation/grpc-1.6/README.md

```diff
@@ -1,5 +1,7 @@
 # Settings for the gRPC instrumentation
 
-| System property                                          | Type    | Default | Description                                         |
-| -------------------------------------------------------- | ------- | ------- | --------------------------------------------------- |
-| `otel.instrumentation.grpc.experimental-span-attributes` | Boolean | `false` | Enable the capture of experimental span attributes. |
+| System property                                             | Type    | Default | Description                                                                               |
+|-------------------------------------------------------------|---------|---------|-------------------------------------------------------------------------------------------|
+| `otel.instrumentation.grpc.experimental-span-attributes`    | Boolean | `false` | Enable the capture of experimental span attributes.                                       |
```

Comment:
```suggestion
| `otel.instrumentation.grpc.experimental-span-attributes`    | Boolean | `false` | Enable the capture all of experimental span attributes.                                       |
``` ?
The reading of the two following properties may be easier.

Reply:
We use the same wording almost everywhere where we have a flag that ends with `experimental-span-attributes`. If you wish to change it it is best to change it everywhere and do it in a separate pr.